amendments.json

```diff
@@ -587,5 +587,21 @@
         "web-platform-tests/wpt#49990"
       ]
     }
+  ],
+  "codec-match-asymmetrical": [
+    {
+      "description": "Compare symmetrical or asymmetrical parameters (not just level-id)",
+      "type": "correction",
+      "status": "candidate",
+      "difftype": "modify",
+      "id": 53,
+      "pr": 3043,
+      "tests": [
+        "webrtc/protocol/av1-profile-asymmetry.https.html"
+      ],
+      "testUpdates": [
+        "web-platform-tests/wpt#0000"
```

Comment:
Before merging: Update to PR that adds webrtc/protocol/av1-profile-asymmetry.https.html

Reply:
Done!